py/torch_tensorrt/dynamo/lowering/_decompositions.py

```diff
@@ -174,6 +174,19 @@ def empty_permuted_decomposition(*args, **kwargs) -> torch.Tensor:
     return torch.empty([empty_size[l] for l in empty_permute], **kwargs).permute(perm)
 
 
+@register_torch_trt_decomposition(
+    torch.ops.aten.scatter_add.default, registry=TORCH_TRT_DECOMPOSITIONS
+)
+def scatter_add_decomposition(
+    input_tensor: torch.Tensor,
+    src_tensor: torch.Tensor,
+    dim: int,
+    index: torch.Tensor,
+) -> torch.Tensor:
+    input_tensor_to_add = torch.scatter(torch.empty_like(input_tensor), dim, index, src_tensor)
```

Comment:
Could `torch.empty_like(input_tensor)` instead be just `input_tensor`, or is it required to be empty?

Reply:
Yes it would be required. The logic is like we would apply scatter with the indices to an empty tensor and then add it to the input tensor.